src/assets/locales/fr.json

```diff
@@ -321,24 +357,25 @@
   "people-sidebar.person-name.you": "Vous",
   "people-sidebar.presence.entering": "Entrée dans la salle",
   "people-sidebar.presence.in-lobby": "Dans le hall d'attente",
-  "people-sidebar.presence.in-room": "Dans la Salle",
+  "people-sidebar.presence.in-room": "Dans la salle",
   "people-sidebar.title": "Personnes ({numPeople})",
```

Comment:
@vincentfretin same here for switching the num and the text "({numPeople}) personnes"